themes/doubletime/doubletime.theme.bash

```diff
@@ -28,45 +28,16 @@ else
     THEME_PROMPT_HOST_COLOR="$blue"
 fi
 
-doubletime_scm_prompt() {
-  CHAR=$(scm_char)
-  if [ $CHAR = $SCM_NONE_CHAR ]; then
-    return
-  elif [ $CHAR = $SCM_GIT_CHAR ]; then
-    echo "$(git_prompt_status)"
-  else
-    echo "[$(scm_prompt_info)]"
-  fi
-}
-
```

Comment:
> tested all of the touched themes to see that they look the same 😄

It appears to me this function doesn't produce the same result as the centralized version of `scm_prompt`.

Reply:
yes, but adding the SCM_CHAR here did not seem like a really big problem, removing it however will make the centralised more complicated

Reply:
OK, if you intentionally reduced the complexity, it is fine!